apimonitor/monitor/health.py

```
@@ -0,0 +1,162 @@
+"""
+Health check implementation with retry and backoff logic.
+"""
+import time
+import logging
+from typing import Dict, Any, Optional
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)